external_server/adapters/api/module_lib.py

```diff
@@ -0,0 +1,153 @@
+import ctypes as ct
+import sys
+from typing import Any
+import os
+import threading
+
+sys.path.append("lib/fleet-protocol/protobuf/compiled/python")
+
+from external_server.models.structures import (
+    Config,
+    Buffer,
+    DeviceIdentification,
+    DisconnectTypes,
+)
+from external_server.models.structures import (
+    Config,
+    KeyValue,
+)
+
+
+def empty_command_buffer() -> Buffer:
+    return Buffer(ct.c_char_p(), 0)
+
+
+def empty_device_identification() -> DeviceIdentification:
+    return DeviceIdentification(
+        0, 0, Buffer(ct.c_char_p(), 0), Buffer(ct.c_char_p()), 0
+    )
+
+
+class ModuleLibrary:
```

Comment:
Provide some docstring. Most importantly it should explain that this class wrapps some external .so library.